src/config.rs

```diff
@@ -306,170 +391,122 @@ fn convert_yaml_map_to_json(
+                assert_eq!(serde_json::Value::Object(settings.0), expected);
+            }
+            _ => panic!("Expected an Individual policy"),
+        }
     }
 
-    #[test]
-    fn handle_yaml_map_with_no_data() {
-        let input = r#"
----
-example:
-  url: file:///tmp/namespace-validate-policy.wasm
-  settings: {}
-"#;
-        let policies: HashMap<String, Policy> = serde_yaml::from_str(input).unwrap();
-        assert!(!policies.is_empty());
-
-        let policy = policies.get("example").unwrap();
-        let json_data = convert_yaml_map_to_json(serde_yaml::Mapping::from_iter(
-            policy
-                .settings
-                .as_ref()
-                .unwrap()
-                .iter()
-                .map(|(key, value)| (serde_yaml::Value::String(key.clone()), value.clone())),
-        ));
-        assert!(json_data.is_ok());
-
-        let settings = json_data.unwrap();
-        assert!(settings.is_empty());
-    }
+    //     #[test]
+    //     fn handle_yaml_map_with_data() {
+    //         let input = r#"
+    // ---
+    // example:
+    //   url: file:///tmp/namespace-validate-policy.wasm
+    //   settings:
+    //     valid_namespace: valid
+    // "#;
+    //         let policies: HashMap<String, Policy> = serde_yaml::from_str(input).unwrap();
+    //         assert!(!policies.is_empty());
+
+    //         let policy = policies.get("example").unwrap();
+    //         let json_data = convert_yaml_map_to_json(serde_yaml::Mapping::from_iter(
+    //             policy
+    //                 .settings
+    //                 .as_ref()
+    //                 .unwrap()
+    //                 .iter()
+    //                 .map(|(key, value)| (serde_yaml::Value::String(key.clone()), value.clone())),
+    //         ));
+    //         assert!(json_data.is_ok());
+
+    //         let settings = json_data.unwrap();
+    //         assert_eq!(settings.get("valid_namespace").unwrap(), "valid");
+    //     }
+
+    //     #[test]
+    //     fn handle_yaml_map_with_no_data() {
+    //         let input = r#"
+    // ---
+    // example:
+    //   url: file:///tmp/namespace-validate-policy.wasm
+    //   settings: {}
+    // "#;
+    //         let policies: HashMap<String, Policy> = serde_yaml::from_str(input).unwrap();
+    //         assert!(!policies.is_empty());
+
+    //         let policy = policies.get("example").unwrap();
+    //         let json_data = convert_yaml_map_to_json(serde_yaml::Mapping::from_iter(
+    //             policy
+    //                 .settings
+    //                 .as_ref()
+    //                 .unwrap()
+    //                 .iter()
+    //                 .map(|(key, value)| (serde_yaml::Value::String(key.clone()), value.clone())),
+    //         ));
+    //         assert!(json_data.is_ok());
+
+    //         let settings = json_data.unwrap();
+    //         assert!(settings.is_empty());
+    //     }
```

Comment:
Cannot we remove this comments ?

Reply:
ouch, good catch! Yes, let's remove these tests since they are not useful